sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -340,46 +467,46 @@ _PI_CL(piDeviceRelease, clReleaseDevice)
 _PI_CL(piextGetDeviceFunctionPointer, OCL(piextGetDeviceFunctionPointer))
 // Context
-_PI_CL(piContextCreate, clCreateContext)
-_PI_CL(piContextGetInfo, clGetContextInfo)
-_PI_CL(piContextRetain, clRetainContext)
-_PI_CL(piContextRelease, clReleaseContext)
+_PI_CL(piContextCreate,     OCL(piContextCreate))
+_PI_CL(piContextGetInfo,    clGetContextInfo)
+_PI_CL(piContextRetain,     clRetainContext)
+_PI_CL(piContextRelease,    clReleaseContext)
 // Queue
 _PI_CL(piQueueCreate, OCL(piQueueCreate))
 _PI_CL(piQueueGetInfo, clGetCommandQueueInfo)
 _PI_CL(piQueueFinish, clFinish)
 _PI_CL(piQueueRetain, clRetainCommandQueue)
 _PI_CL(piQueueRelease, clReleaseCommandQueue)
 // Memory
-_PI_CL(piMemBufferCreate, clCreateBuffer)
-_PI_CL(piMemImageCreate, clCreateImage)
-_PI_CL(piMemGetInfo, clGetMemObjectInfo)
-_PI_CL(piMemImageGetInfo, clGetImageInfo)
-_PI_CL(piMemRetain, clRetainMemObject)
-_PI_CL(piMemRelease, clReleaseMemObject)
-_PI_CL(piMemBufferPartition, clCreateSubBuffer)
+_PI_CL(piMemBufferCreate,    OCL(piMemBufferCreate))
+_PI_CL(piMemImageCreate,     OCL(piMemImageCreate))
+_PI_CL(piMemGetInfo,         clGetMemObjectInfo)
+_PI_CL(piMemImageGetInfo,    clGetImageInfo)
+_PI_CL(piMemRetain,          clRetainMemObject)
+_PI_CL(piMemRelease,         clReleaseMemObject)
+_PI_CL(piMemBufferPartition, OCL(piMemBufferPartition))
 // Program
-_PI_CL(piProgramCreate, OCL(piProgramCreate))
-_PI_CL(piclProgramCreateWithSource, clCreateProgramWithSource)
-_PI_CL(piclProgramCreateWithBinary, clCreateProgramWithBinary)
-_PI_CL(piProgramGetInfo, clGetProgramInfo)
-_PI_CL(piProgramCompile, clCompileProgram)
-_PI_CL(piProgramBuild, clBuildProgram)
-_PI_CL(piProgramLink, clLinkProgram)
-_PI_CL(piProgramGetBuildInfo, clGetProgramBuildInfo)
-_PI_CL(piProgramRetain, clRetainProgram)
-_PI_CL(piProgramRelease, clReleaseProgram)
+_PI_CL(piProgramCreate,             OCL(piProgramCreate))
+_PI_CL(piclProgramCreateWithSource, OCL(piclProgramCreateWithSource))
+_PI_CL(piclProgramCreateWithBinary, OCL(piclProgramCreateWithBinary))
+_PI_CL(piProgramGetInfo,            clGetProgramInfo)
+_PI_CL(piProgramCompile,            clCompileProgram)
+_PI_CL(piProgramBuild,              clBuildProgram)
+_PI_CL(piProgramLink,               OCL(piProgramLink))
+_PI_CL(piProgramGetBuildInfo,       clGetProgramBuildInfo)
+_PI_CL(piProgramRetain,             clRetainProgram)
+_PI_CL(piProgramRelease,            clReleaseProgram)
 // Kernel
-_PI_CL(piKernelCreate, clCreateKernel)
-_PI_CL(piKernelSetArg, clSetKernelArg)
-_PI_CL(piKernelGetInfo, clGetKernelInfo)
-_PI_CL(piKernelGetGroupInfo, clGetKernelWorkGroupInfo)
+_PI_CL(piKernelCreate,          OCL(piKernelCreate))
+_PI_CL(piKernelSetArg,          clSetKernelArg)
+_PI_CL(piKernelGetInfo,         clGetKernelInfo)
+_PI_CL(piKernelGetGroupInfo,    clGetKernelWorkGroupInfo)
 _PI_CL(piKernelGetSubGroupInfo, clGetKernelSubGroupInfo)
 _PI_CL(piKernelRetain, clRetainKernel)
 _PI_CL(piKernelRelease, clReleaseKernel)
```

Comment:
This is an insignificant nitpicking, but why did you align some strings while ignoring the other?

Reply:
no worries. It is a valid question.
I must have used clang-format on those select lines. I'll  make it more consistent.

Reply:
This is still unresolved...

Reply:
done in the new commit.